guides/assets/stylesrc/_main.scss

```diff
@@ -795,4 +824,27 @@ body.guide {
     }
   }
   
-} // body.guide
\ No newline at end of file
+} // body.guide
+
+html.translated-rtl:lang("ar", "fa") {
+  body.guide {
+    h1, h2, h3, h4, h5, h6, p, dl, dd, dt, ul, ol, li {
+      font-family: "Noto Sans Arabic", sans-serif;
+      font-optical-sizing: auto;
+      font-weight: 400;
+      font-style: normal;
+      font-variation-settings: "wdth" 100;
+    }
+  }
+}
+
+html.translated-rtl:lang("he", "iw") {
```

Comment:
Let's just use the `lang`, or rather a combination of `dir` and `lang`, and avoid `.translated-*` in the css.

Reply:
Addressed in c0220852